doc-content/enterprise-only/integration/dev-studio-plug-in-install-proc.adoc

```diff
@@ -1,24 +1,25 @@
 [id='dev-studio-plug-in-install-proc']
-= Installing Red Hat JBoss Developer Studio plug-ins
+= Installing Red Hat Developer Studio plug-ins
 
-The Red Hat JBoss Developer Studio plug-ins for {PRODUCT}  are based on the community version of the plug-ins. For this reason, the {PRODUCT} plug-in is called the Drools plug-in.
```

Comment:
Change to:
For this reason, the {PRODUCT} plug-ins are called the jBPM and Drools plug-ins.
endif::[]
ifdef::DM[]
For this reason, the {PRODUCT} plug-in is called the Drools plug-in.
endif::[]

This is from the Planning guide and was sign-off on by Eng.

Reply:
Done, added PAM to the ifdef